src/main/java/graphql/GraphQLConfiguration.java

```diff
@@ -0,0 +1,150 @@
+package graphql;
+
+import graphql.parser.ParserOptions;
+
+/**
+ * This allows you to control specific aspects of the GraphQL system
+ * including some JVM wide settings and some per execution settings
+ * as well as experimental ones
+ */
+public class GraphQLConfiguration {
+    static final GraphQLConfiguration INSTANCE = new GraphQLConfiguration();
+    static final ParserCfg PARSER_CFG = new ParserCfg();
+    static final IncrementalSupportCfg INCREMENTAL_SUPPORT_CFG = new IncrementalSupportCfg();
+
+    private GraphQLConfiguration() {
+    }
+
+    public ParserCfg parser() {
+        return PARSER_CFG;
+    }
+
+    @ExperimentalApi
+    public IncrementalSupportCfg incrementalSupport() {
+        return INCREMENTAL_SUPPORT_CFG;
+    }
+
+    public static class ParserCfg {
```

Comment:
parse config here is repeated - so sure you can go direct to ParserOptions and thats not too bad a name to discover

BUT

This makes it central and future ones can be here